src/uu/cut/cut.md

```diff
@@ -13,80 +15,90 @@ a sequence (which columns to print), and provide a data source
 
-    If sourcefile arguments are specified, stdin is ignored and all files are
-    read in consecutively if a sourcefile is not successfully read, a warning
-    will print to stderr, and the eventual status code will be 1, but cut
-    will continue to read through proceeding sourcefiles
+If sourcefile arguments are specified, stdin is ignored and all files are
+read in consecutively if a sourcefile is not successfully read, a warning
+will print to stderr, and the eventual status code will be 1, but cut
+will continue to read through proceeding sourcefiles
 
-    To print columns from both STDIN and a file argument, use - (dash) as a
-    sourcefile argument to represent stdin.
+To print columns from both STDIN and a file argument, use - (dash) as a
+sourcefile argument to represent stdin.
+```
 
 Field Mode options
 
-    The fields in each line are identified by a delimiter (separator)
-
-    Set the delimiter
-        Set the delimiter which separates fields in the file using the
-        --delimiter (-d) option. Setting the delimiter is optional.
-        If not set, a default delimiter of Tab will be used.
-
-        If the -w option is provided, fields will be separated by any number
-        of whitespace characters (Space and Tab). The output delimiter will
-        be a Tab unless explicitly specified. Only one of -d or -w option can be specified.
-        This is an extension adopted from FreeBSD.
-
-    Optionally Filter based on delimiter
-        If the --only-delimited (-s) flag is provided, only lines which
-        contain the delimiter will be printed
-
-    Replace the delimiter
-        If the --output-delimiter option is provided, the argument used for
-        it will replace the delimiter character in each line printed. This is
-        useful for transforming tabular data - e.g. to convert a CSV to a
-        TSV (tab-separated file)
+```
+The fields in each line are identified by a delimiter (separator)
+
+Set the delimiter
+    Set the delimiter which separates fields in the file using the
+    --delimiter (-d) option. Setting the delimiter is optional.
+    If not set, a default delimiter of Tab will be used.
+
+    If the -w option is provided, fields will be separated by any number
+    of whitespace characters (Space and Tab). The output delimiter will
+    be a Tab unless explicitly specified. Only one of -d or -w option can be specified.
+    This is an extension adopted from FreeBSD.
+
+Optionally Filter based on delimiter
+    If the --only-delimited (-s) flag is provided, only lines which
+    contain the delimiter will be printed
+
+Replace the delimiter
+    If the --output-delimiter option is provided, the argument used for
+    it will replace the delimiter character in each line printed. This is
+    useful for transforming tabular data - e.g. to convert a CSV to a
+    TSV (tab-separated file)
+```
 
 Line endings
 
+```
     When the --zero-terminated (-z) option is used, cut sees \\0 (null) as the
```

Comment:
When I've removed indent here the output looks like:

```

The fields in each line are identified by a delimiter (separator)

Set the delimiter
    Set the delimiter which separates fields in the file using the
    --delimiter (-d) option. Setting the delimiter is optional.
    If not set, a default delimiter of Tab will be used.

    If the -w option is provided, fields will be separated by any number
    of whitespace characters (Space and Tab). The output delimiter will
    be a Tab unless explicitly specified. Only one of -d or -w option can be specified.
    This is an extension adopted from FreeBSD.

Optionally Filter based on delimiter
    If the --only-delimited (-s) flag is provided, only lines which
    contain the delimiter will be printed

Replace the delimiter
    If the --output-delimiter option is provided, the argument used for
    it will replace the delimiter character in each line printed. This is
    useful for transforming tabular data - e.g. to convert a CSV to a
    TSV (tab-separated file)


Line endings


When the --zero-terminated (-z) option is used, cut sees \\0 (null) as the
'line ending' character (both for the purposes of reading lines and
separating printed lines) instead of \\n (newline). This is useful for
tabular data where some of the cells may contain newlines

echo 'ab\\0cd' | cut -z -c 1
will result in 'a\\0c\\0'
```

I'm not sure if there is a better approach for this case.